pkg/config/env/ssl.go

```diff
@@ -0,0 +1,13 @@
+package env
+
+import (
+	"os"
+)
+
+// ProxyForceSSL returns true if the PROXY_FORCE_SSL
+// env is set to "true", otherwise defaults to false.
+// This is used to make sure the Proxy would redirect
+// to https on any request.
+func ProxyForceSSL() bool {
+	return os.Getenv("PROXY_FORCE_SSL") == "true"
```

Comment:
can we name this ATHENS_FORCE_SSL? with the env conf change it makes it easier to map to config values

Reply:
@rchakra3 this is potentially *not* an option for Olympus, because I imagine Olympus is going to be public and with https by default.

Reply:
ah, that makes sense!